lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/CLI/Server.hs

```diff
@@ -159,6 +159,20 @@ spec = do
             --
             -- but in practice, we only have INFO logs on start-up.
 
+        it "LOGGING - Serve shuts down logging correctly" $ \ctx -> do
+            let args =
+                    ["serve"
+                    , "--node-port"
+                    , show (ctx ^. typed @(Port "node"))
+                    , "--random-port"
+                    , "--verbose"
+                    , "--genesis-block-hash"
+                    , block0H
+                    ]
+            let process = proc' (commandName @t) args
+            (out, _) <- collectStreams (10, 0) process
```

Comment:
Hi @rvl. Are you able to say a little bit about where `10` and `0` came from here?

Reply:
These are the number of output lines to collect before finishing the test.
The `collectStreams` function is a bit fishy actually.